dir.traversal.targets

```diff
@@ -86,6 +86,32 @@
              ContinueOnError="ErrorAndContinue" />
   </Target>
 
+  <Target Name="ProducesPackageId"
+          Returns="@(PackageIds)" 
```

Comment:
> is it worth just getting the list from the produced packages in the output?

I think the repo API spec's not clear about this, if a full build can be a prereq for `produces` or if it's expected to work on a clean repo. From this PR I was thinking we're going for something build-independent but I don't know if that's realistic. Another wrinkle is that the spec shows package versions in the `produces` json.